package.json

```
@@ -14,7 +14,8 @@
     "lint:js:fix": "eslint config src --fix",
     "lint:sass": "stylelint 'src/**/*.scss' --config .stylelintrc.json",
     "patch:hosts": "fec patch-etc-hosts",
-    "start": "fec dev",
+    "start": "PROXY=true ROUTES_PATH=/routes.config.js fec dev",
```

Moving to config dir will help here as well, it looks weird to start relative path with `/`
```suggestion
    "start": "PROXY=true ROUTES_PATH=config/routes.config.js fec dev",
```